debian/wb-configs.postinst

```diff
@@ -268,6 +270,10 @@ mosquitto_fixes()
         sed -i 's#pid_file /var/run/mosquitto.pid#pid_file /run/mosquitto/mosquitto.pid#' $MOSQUITTO_CONF
         restart_required=1
     fi
+    if ! grep -q -F "include_dir /usr/share/wb-configs/mosquitto" $MOSQUITTO_CONF; then
+        sed -i '/include_dir \/etc\/mosquitto\/conf.d/iinclude_dir \/usr\/share\/wb-configs\/mosquitto' $MOSQUITTO_CONF
```

Comment:
```suggestion
        sed -i '\#include_dir /etc/mosquitto/conf.d#iinclude_dir /usr/share/wb-configs/mosquitto' $MOSQUITTO_CONF
```

https://backreference.org/2010/02/20/using-different-delimiters-in-sed/index.html